components/automate-es-gateway/habitat/config/nginx.conf

```
@@ -60,7 +60,9 @@ http {
 {{/if~}}
     keepalive 8;
   }
-
+  {{~#if cfg.ngx.http.proxy_ssl_server_name}}
+    proxy_ssl_server_name on;
+  {{~else}}
```

```suggestion
  {{/if~}}
```
Thanks for your PR
It should be a closing "if" right ? @mskdenigma could you please make this change.

---

thanks for catching this, I have fixed it.

---

Thanks